glib/src/param_spec.rs

```diff
@@ -826,6 +942,13 @@ impl ParamSpecEnum {
     }
 }
 
+define_builder!(
+    ParamSpecEnum,
+    ParamSpecEnumBuilder,
+    enum_type: crate::Type = crate::Type::ENUM,
```

Comment:
The enum type must be a concrete enum type and is mandatory, so must be given in the `new()` / `builder()` function